python/paddle/v2/fluid/framework.py

```diff
@@ -418,29 +418,28 @@ def find_name(var_list, name):
                     return True
             return False
 
-        if inputs is not None:
```

Comment:
Is this assuming inputs is not possible to be None? If so maybe we need to remove the default None value from the argument list.